### ISSUE TITLE
ReferenceError: messageId is not defined on sendImageAsStickerAsReply even when messageId is specified

### ISSUE THREAD

**Issue by tcortega** [open-wa/wa-automate-nodejs#1004]
**Describe the bug**
I'm trying to use sendImageAsStickerAsReply, and it's giving me an error that it shouldn't.

**Steps to Reproduce**
Attempting to use the function.

**create() code**
This is the code you use to create the client. e.g
```javascript
create({
        headless: headless,
        qrRefreshS: 20,
        qrTimeout: 0,
        authTimeout: 0,
        autoRefresh: true,
        restartOnCrash: start,
        cacheEnabled: false,
        logConsole: true,
        useChrome: true,
        killProcessOnBrowserClose: true,
        throwErrorOnTosBlock: false
    }).then((client) => start(client))
    .catch((err) => new Error(err))
```

**Expected behavior**
Should give an error of `ReferenceError: messageId is not defined`

**DEBUG INFO**
This is the info printed to the console when you start your app. It should look like this
```
Debug Info {
WA_VERSION: '2.2043.8',
PAGE_UA: 'WhatsApp/2.2037.6 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.83 Safari/537.36',
WA_AUTOMATE_VERSION: '3.0.6',
BROWSER_VERSION: 'HeadlessChrome/87.0.4272.0'                 
}
```

**Screenshots**
No need

**Host (please complete the following information):**
 - OS: Windows 10 Pro v2004. OS Compilation version is 1904.572

**Additional context**
I bought a license key, and it should be working. I also can't use other features such as tageveryone(), it gives me no errors on console tho. And `id` that is specified on the javascript code, is a property of message.

Here is the code I'm using to make use of this function:
```javascript
const encryptMedia = isQuotedImage ? quotedMsg : message
const _mimetype = isQuotedImage ? quotedMsg.mimetype : mimetype
const mediaData = await decryptMedia(encryptMedia, uaOverride)
const imageBase64 = `data:${_mimetype};base64,${mediaData.toString('base64')}`
client.sendImageAsStickerAsReply(from, imageBase64, id)
```

**Comment by smashah** [comment]
Can you log the value of id please, also, you haven't set your licenseKey in the config object. https://open-wa.github.io/wa-automate-nodejs/interfaces/configobject.html#licensekey

**Comment by tcortega** [comment]
> Can you log the value of id please, also, you haven't set your licenseKey in the config object. https://open-wa.github.io/wa-automate-nodejs/interfaces/configobject.html#licensekey

Did not now about this, and the logged id is:
`false_559189207611-1549075401@g.us_3EB0689DECF51C1C9671`

**Comment by smashah** [comment]
@tcortega can you share your the create logs, please

**Comment by tcortega** [comment]
> @tcortega can you share your the create logs, please

Where can I find those?

**Comment by smashah** [comment]
It's the log output when you run the code

**Comment by tcortega** [comment]
> It's the log output when you run the code

```
ConsoleMessage {tches
  _type: 'error',
  _text: 'Failed to load resource: net::ERR_FAILED',
  _args: [],
  _stackTraceLocations: [
    {
      url: 'https://crashlogs.whatsapp.net/wa_fls_upload_check?type=crashlog&access_token=1063127757113399%7C745146ffa34413f9dbb5469f5370b7af',
      lineNumber: undefined
    }
  ]
}
ConsoleMessage {tches
  _type: 'error',
  _text: `Refused to connect to 'https://api.ipify.org/?format=json' because it violates the following Content Security Policy directive: "connect-src 'self' data: blob: https://*.whatsapp.net https://www.facebook.com https://*.giphy.com https://*.tenor.co https://crashlogs.whatsapp.net/wa_clb_data https://crashlogs.whatsapp.net/wa_fls_upload_check https://www.bingapis.com/api/v6/images/search https://*.google-analytics.com wss://*.web.whatsapp.com wss://web.whatsapp.com https://dyn.web.whatsapp.com".\n`,
  _args: [],
  _stackTraceLocations: [ { url: undefined, lineNumber: 1 } ]
}
```

This?

**Comment by smashah** [comment]
@tcortega this will be fixed today.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
@tcortega Should be fixed in [v3.0.7](https://github.com/open-wa/wa-automate-nodejs/blob/master/CHANGELOG.md#v307---2020-11-01)

**Comment by tcortega** [comment]
> @tcortega Should be fixed in [v3.0.7](https://github.com/open-wa/wa-automate-nodejs/blob/master/CHANGELOG.md#v307---2020-11-01)

Checked and it's working as expected! 👍
